backend/initial.json

```diff
@@ -0,0 +1,778 @@
+[
```

Comment:
I would prefer a lot for readability and maintainability reasons if we had e.g. a management command `init_project` that created these objects to the database instead of a hardcoded json file

Reply:
hmm, I think that might be a good idea but not sure if there's a big difference between `manage.py init_project` vs `manage.py loaddata initial.json`. I left this like this until we see how the migration efforts will change the playground.